docs/application/web/api/6.5/device_api/tv/tizen/ml_pipeline.html

```diff
@@ -0,0 +1,2120 @@
+For more information about pipeline description, see the <a href="https://gstreamer.freedesktop.org/documentation/tools/gst-launch.html?gi-language=c#pipeline-description">Pipeline Description</a> section of gst-launch-1.0 documentation.
+            </p>
+           </div>
+<p><span class="remark">Remark: </span>
+ Pipeline creation is an asynchronous process, during which the
+pipeline state transitions through <var>READY</var> to <var>PAUSED</var>.
+The pipeline is ready to use when it is in <var>PAUSED</var> state.
+            </p>
+<div class="parameters">
+<p><span class="param">Parameters:</span></p>
+<ul>
+          <li class="param">
+<span class="name">description</span>:
+ A description of the pipeline.
+                </li>
+          <li class="param">
+<span class="name">listener</span><span class="optional"> [optional]</span><span class="optional"> [nullable]</span>:
+ Listener function that will be called when pipeline state changes.
+                </li>
+        </ul>
+</div>
+<div class="returntype">
+<p><span class="return">Return value:</span></p>
+<ul>
+<span class="return">Pipeline:</span>
+ A newly created <a href="#Pipeline">Pipeline</a> object.
+              </ul>
+</div>
+<div class="exceptionlist">
+<p><span class="except">Exceptions:</span></p>
+          <ul class="exception"><li>WebAPIException<ul>
+<li class="list"><p>
+ with error type NotSupportedError, if the feature is not supported.
+                </p></li>
+<li class="list"><p>
+ with error type SecurityError, if the application does not have the privilege to call this method.
+                </p></li>
+<li class="list"><p>
+ with error type TypeMismatchError, if any input attribute is not compatible with the expected type for this attribute.
+                </p></li>
+<li class="list"><p>
+ with error type AbortError, if the <em>description</em> is invalid or any other error occurs.
+                </p></li>
+</ul>
+</li></ul>
+        </div>
+<div class="example">
+<span class="example"><p>Code example:</p></span><pre name="code" class="examplecode prettyprint">var pipelineDescription = "videotestsrc ! tizenwlsink";
+function pipelineStateChangeListener(newState)
+{
+  console.log("New pipeline state: " + newState);
+}
+
+var pipeline = tizen.ml.pipeline.createPipeline(pipelineDescription, pipelineStateChangeListener);
+/* after use */
+pipeline.dispose();
+</pre>
+</div>
+<div class="output">
+<span class="title"><p>Output example:</p></span><pre>New pipeline state: READY
+New pipeline state: PAUSED
+</pre>
+</div>
+</dd>
+<dt class="method" id="MachineLearningPipeline::registerCustomFilter">
+<a class="backward-compatibility-anchor" name="::Pipeline::MachineLearningPipeline::registerCustomFilter"></a><code><b><span class="methodName">registerCustomFilter</span></b></code>
+</dt>
+<dd>
+<div class="brief">
+ Registers a <a href="#CustomFilter">CustomFilter</a> implementing custom transform of the data coming through the pipeline.
```

Comment:
```suggestion
 Registers a <a href="#CustomFilter">CustomFilter</a>, which implements custom transform to the data coming through the pipeline.
```
check for technical correctness